pages/api/subscription/index.ts

```diff
@@ -0,0 +1,26 @@
+import { axios } from '../../../src/utils';
+
+export default async function handler(req, res) {
```

Comment:
We can type these parameters

```suggestion
import { NextApiRequest, NextApiResponse } from 'next';
import { axios } from '../../../src/utils';

export default async function handler(
  req: NextApiRequest,
  res: NextApiResponse,
) {
```